test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -71,6 +71,15 @@ Describe 'Classes inheritance syntax' -Tags "CI" {
         $getter.Attributes -band [System.Reflection.MethodAttributes]::Virtual |Should -Be ([System.Reflection.MethodAttributes]::Virtual)
     }
 
+    It 'can implement inherited .NET interface properties' {
+        Add-Type -TypeDefinition 'public interface IParent { int ParentInteger { get; set; } }
+                                  public interface IChild : IParent { int ChildInteger { get; set; } }'
+        $C1 = Invoke-Expression 'class ClassWithInheritedInterfaces : IChild { [int]$ParentInteger; [int]$ChildInteger } [ClassWithInheritedInterfaces]::new()'
+        $getter = $C1.GetType().GetMember('get_ParentInteger')
```

Comment:
nit: Is creating an object of `ClassWithInheritedInterfaces` needed? Can you just make the `Invoke-Expression` return the type?

Reply:
Yeah, absolutely, fixed now